src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js

```diff
@@ -120,28 +115,6 @@ module.exports = {
       $ref: 'output/paragraph-list_of_link_teasers',
     },
     fieldPressReleaseBlurb: { $ref: 'ProcessedString' },
-    fieldLeadership: {
-      type: 'array',
-      items: {
-        entity: partialSchema(personProfileSchema, [
-          'entityPublished',
-          'title',
-          'fieldNameFirst',
-          'fieldLastName',
-          'fieldSuffix',
-          'fieldEmailAddress',
-          'fieldPhoneNumber',
-          'fieldDescription',
-          'fieldOffice',
-          'fieldIntroText',
-          'fieldPhotoAllowHiresDownload',
-          'fieldMedia',
-          'fieldBody',
-          'changed',
-          'entityUrl',
-        ]),
-      },
-    },
```

Comment:
Why is this being removed?

Reply:
Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh